mmv1/third_party/terraform/services/compute/resource_compute_instance_group_manager.go.erb

```diff
@@ -216,6 +216,48 @@ func ResourceComputeInstanceGroupManager() *schema.Resource {
 				},
 			},
 
+			<% unless version == "ga" -%>
```

Comment:
can we indent it similarly to the other fields?

Reply:
Done.